src/components/field-key/new.vue

```diff
@@ -182,11 +182,14 @@ export default {
     width: 80%;
     margin-bottom: 0px;
   }
+
+  + .field-key-qr-panel {
+    margin: 15px auto 30px;
+  }
 }
 
 .field-key-qr-panel {
   box-shadow: $box-shadow-popover;
```

Comment:
I think this box shadow should be moved above as well. Otherwise, it will be applied to the `FieldKeyQrPanel` shown in the popover. When I look at the [component](https://github.com/getodk/central-frontend/blob/v2023.5.0/src/components/field-key/new.vue) in v2023.5, I see that the box shadow is nested under `#field-key-new-success` and not applied to the popover. The styles here aren't automatically scoped to the `FieldKeyNew` component, so specifying a style on `.field-key-qr-panel` outside of `#field-key-new-success` will apply to all `.field-key-qr-panel` elements.